src/Uno.UWPSyncGenerator/Properties/launchSettings.json

```diff
@@ -2,7 +2,7 @@
   "profiles": {
     "Uno.UWPSyncGenerator": {
       "commandName": "Project",
-      "commandLineArgs": "sync"
+      "commandLineArgs": "doc"
```

Comment:
```suggestion
      "commandLineArgs": "sync"
```

Reply:
My mistake. It was in the wrong branch. The other is still in progress.
Reverted..